src/screens/Menu/WalletSettings.js

```diff
@@ -111,6 +117,7 @@ class WalletSettings extends React.Component<Props, State> {
       this.setState({ supportedBiometryType });
     });
     this.retrieveWalletObject();
+    isRecoveryPortalDisabled = firebaseRemoteConfig.getBoolean(REMOTE_CONFIG.RECOVERY_PORTAL_DISABLED);
```

Comment:
Why `disabled` instead of `enabled`? If you look in `Services`, we tend to use `enabled` for feature flags everywhere (e.g. `isPoolTogetherEnabled`, `isSablierEnabled`...)

Reply:
because the task specified the flag to be `disabled` https://linear.app/pillarproject/issue/PIL-837/wrap-recovery-portal-access-with-feature-flag